src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/ExternalMethodImport.cs

```diff
@@ -39,5 +39,11 @@ public class ExternalMethodImport : DelayLoadHelperImport, IMethodNode
         public MethodDesc Method => _method.Method;
 
         public override int ClassCode => 458823351;
+
+        // This is just here in case of future extension (everything is already compared in the base CompareToImpl)
+        public override int CompareToImpl(ISortableNode other, CompilerComparer comparer)
+        {
+            return base.CompareToImpl(other, comparer);
```

Comment:
Another case where there are fields that are not used in this class. We should delete them. Having a compare function that doesn't take them into account is just confusing.